gdal/ogr/ogrsf_frmts/mitab/mitab_feature.cpp

```diff
@@ -1558,6 +1558,27 @@ const char *TABFontPoint::GetStyleString() const
     return m_pszStyleString;
 }
 
+void TABFontPoint::SetSymbolFromStyle(OGRStyleSymbol* poSymbolStyle)
+{
+    ITABFeatureSymbol::SetSymbolFromStyle(poSymbolStyle);
+
+    GBool bIsNull = 0;
+
+    // Try to set font glyph number
+    const char* pszSymbolId = poSymbolStyle->Id(bIsNull);
+    if((!bIsNull) && pszSymbolId && strstr(pszSymbolId, "font-sym-"))
+    {
+        const int nSymbolId = atoi(pszSymbolId+9);
```

Comment:
this assumes that the pszSymbolId starts with "font-sym-", but strstr() tests only for its existence. So either use STARTS_WITH(), or take the pointer returned by strstr()